### ISSUE TITLE
NoMethodError: undefined method `find_plugin' for module OpenapiFirst

### ISSUE THREAD

**Issue by gobijan** [ahx/openapi_first#269]
Hi Andreas,
hope you're doing well. Thanks for your continuous work on this gem of a gem.
I updated today to 2.0.

The configuration seems to be broken now after your latest changes (removal of plugin system commit?).

I get a no_method error:

```
NoMethodError: undefined method `find_plugin' for module OpenapiFirst (NoMethodError)
OpenapiFirst.find_plugin(:default)::ErrorResponse

```

Caused by the config as given in the readme.

```
OpenapiFirst.configure do |config|
  # Specify which plugin is used to render error responses returned by the request validation middleware (defaults to :default)
  config.request_validation_error_response = :jsonapi
  # Configure if the request validation middleware should raise an exception (defaults to false)
  # config.request_validation_raise_error = true
end

```

When I comment out `config.request_validation_error_response = :jsonapi` it works again though the request validation response format is not set correctly.

What is the way to go forward?

**Comment by ahx** [comment]
Thanks so much for the report and for your interest in this project @gobijan !

This should be solved by https://github.com/ahx/openapi_first/issues/269

**Comment by ahx** [comment]
A fix has been released in 2.0.1. Thanks again for the report.
I will  have some free time from August till November and I want to work on this outside of work. So don't hesitate to ask for new features or highlight any annoyances (like obviously not having sufficient test coverage for the configuration part).

**Comment by gobijan** [comment]
Fixed :)

**Comment by ahx** [comment]
Okay. I just released 2.0.2. Sorry. I will take some time next time.
